src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    /*
+    //First time I guessed that it will be a great idea, but after reading a checklist
+    //I found it useless. I will let it lay here, just in case it may become helpful once
+
+    public Figure[] generateRandomFigureList(Figure[] figures) {
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = getRandomFigure();
+        }
+        for (int i = figures.length / 2; i < figures.length; i++) {
+            figures[i] = getDefaultFigure();
+        }
```

Comment:
we should move this code to the main() method and create objects in for loop instead of manual creation